src/Core/include/OpenLoco/Core/EnumFlags.hpp

```diff
@@ -39,6 +39,11 @@ namespace OpenLoco::Core
     inline constexpr ENM operator~(const ENM a) noexcept                                                                    \
     {                                                                                                                       \
         return static_cast<ENM>(~static_cast<std::underlying_type_t<ENM>>(a));                                              \
+    }                                                                                                                       \
+    inline constexpr ENM operator-=(ENM& a, const ENM b) noexcept                                                           \
```

Comment:
What was needing this? Perhaps we can refactor it. I'm not sure if we should add it it feels like a weird thing to do with a flag.